editor/test/integration/unload_test.clj

```diff
@@ -544,22 +544,21 @@
               (is (not (loaded-proj-path? "/unloaded/unloaded.tilesource")))
               (is (loaded-proj-path? "/loaded_referencing_unloaded_tilesource.script")))
 
-            (testing "Editing a script to reference unloaded Lua modules will load transitive dependencies."
+            (testing "Editing a script to reference unloaded Lua modules will not load transitive dependencies."
               (let [node-load-info-tx-data-calls
                     (call-logged-transact!
                       (test-util/set-code-editor-lines
                         (project/get-resource-node project "/loaded_referencing_unloaded_lua.script")
                         ["local first_alias = require 'unloaded.unloaded'"
                          "local second_alias = require 'unloaded.unloaded'"]))]
 
-                (is (= ["/unloaded/unloaded.lua"
-                        "/unloaded/unloaded_math.lua"]
+                (is (= []
```

Comment:
We no longer connect completions, and module deps are dynamic, so it seems unloaded Lua modules are no longer loaded.